### PR TITLE
Pull uuid lambda into new helper method.

### PR DESCRIPTION
**Pull request by justinabrahms**
This allows django to serialize it for the purposes of migration generation.

### CODE REVIEW

File: webhooks/migrations/0002_auto_20151208_1846.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (noisecapella) [inline webhooks/migrations/0002_auto_20151208_1846.py]: Can you rename this migration to something human readable?

**Comment by noisecapella** [comment]
This looks fine overall but can you explain what Django is doing that treats named functions differently from lambdas in this case (if you know)?

**Comment by justinabrahms** [comment]
https://docs.djangoproject.com/en/1.9/ref/models/fields/#default says that lambdas are incompatible with migrations. If you look at the resulting migration, it references it via import, which can't happen if it's a lambda.

**Comment by noisecapella** [comment]
Sounds good :+1:

**Pull request merged**